packages/react-components/react-nav-preview/src/components/NavCategory/NavCategory.tsx

```diff
@@ -1,22 +1,24 @@
 import * as React from 'react';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
-// import { useCustomStyleHook_unstable } from '@fluentui/react-shared-contexts';
 import { useNavCategory_unstable } from './useNavCategory';
 import { renderNavCategory_unstable } from './renderNavCategory';
 import { useNavCategoryStyles_unstable } from './useNavCategoryStyles.styles';
 import type { NavCategoryProps } from './NavCategory.types';
+import { useNavCategoryContextValues_unstable } from '../useNavCategoryContextValues_unstable';
```

Comment:
nit, we try to keep type imports at the bottom

Reply:
Ahh - good to know. I'll update this file, and try to keep that in mind going forward.

I don't want to do a thorough pass through all files in this package just because it would likely touch every file. I'll make updates for this as I'm making more meaningful changes over the course of development.